src/inference/hmc.jl

```diff
@@ -470,23 +399,19 @@ function steps!(
     vi,
     samples;
     rng::AbstractRNG=GLOBAL_RNG,
-    verbose::Bool=true,
-    progress::Bool=false
+    verbose::Bool=true
 )
-    ahmc_samples, stats = AHMC.sample(
+    ahmc_samples = AHMC.sample(
         rng,
         spl.info[:h],
         spl.info[:traj],
         Vector{Float64}(vi[spl]),
         spl.alg.n_iters;
-        verbose=verbose,
-        progress=progress
```

Comment:
I believe this is a merge issue, we should not drop progress. Same for all places

Reply:
Progress is now handled by the `callback!` function in the interface, which each sampler should dispatch on separately. No sampler should handle progress meters internally.